src/containers/Firebase/firebase.js

```diff
@@ -0,0 +1,20 @@
+import app from 'firebase/app';
+
+const firebaseConfig = {
+  apiKey: process.env.REACT_APP_API_KEY,
+  authDomain: process.env.REACT_APP_AUTH_DOMAIN,
+  databaseURL: process.env.REACT_APP_DATABASE_URL,
+  projectId: process.env.REACT_APP_PROJECT_ID,
+  storageBucket: process.env.REACT_APP_STORAGE_BUCKET,
+  messagingSenderId: process.env.REACT_APP_MESSAGING_SENDER_ID,
+  appId: process.env.REACT_APP_APP_ID,
+  measurementId: process.env.REACT_APP_MEASURMENT_ID,
+};
+
+class Firebase {
+  constructor() {
+    app.initializeApp(firebaseConfig);
+  }
+}
+
+export { Firebase as default };
```

Comment:
Why not export default?
```suggestion
export default Firebase;
```